lib/index.d.ts

```diff
@@ -13,7 +13,9 @@ export default class SQS {
     log(message: string, data?: Record<string, any>): void;
     success(message: string, data?: Record<string, any>): void;
     error(err: Error, data?: Record<string, any>): void;
-    init(): Promise<SQS>;
+    private processQueueUrls;
+    private listQueuesRecursively;
+    init(queueNamePrefix?: string): Promise<SQS>;
```

Comment:
can this  queueNamePrefix be extended to a list possibly?

Reply:
Currently, the AWS library function that gets triggered underneath only expects a string prefix... 
Converting it to a list of prefixes... will require extra effort.
And as of now, we don't have such a use case where we want to maintain the list of only specific queues only.

Once that would be the requirement, surely can be picked up in some way or form.